crates/gpui/src/platform/cosmic_text/text_system.rs

```diff
@@ -69,8 +69,10 @@ impl PlatformTextSystem for CosmicTextSystem {
             .font_system
             .db()
             .faces()
-            .map(|face| face.post_script_name.clone())
-            .collect()
+            .filter_map(|face| face.families.first().map(|family| family.0.clone()))
+            .collect::<BTreeSet<String>>()
```

Comment:
Rather than collecting into a BTreeSet, and allocating the whole thing twice, could we use a `Vec::sort` and `Vec::dedup`?

Reply:
On macOS, BTreeSet was used, so I followed suit and used BTreeSet as well. However, your suggestion is correct; we should avoid multiple memory allocations.

Reply:
Oh, good to know!